storage/index/types.go

```diff
@@ -21,12 +21,14 @@
 package index
 
 import (
+	"context"
```

Comment:
Probably want the M3DB context here yeah?

Reply:
yep just wanted to put up a quick diff in case this was blocking you. will cleanup and implement soon.